common/djangoapps/third_party_auth/settings.py

```diff
@@ -24,12 +24,6 @@ def apply_settings(django_settings):
     # Params not in this whitelist will be silently dropped.
     django_settings.FIELDS_STORED_IN_SESSION = _FIELDS_STORED_IN_SESSION
 
-    # Register and configure python-social-auth with Django.
-    django_settings.INSTALLED_APPS += (
-        'social.apps.django_app.default',
-        'third_party_auth',
-    )
-
```

Comment:
I understand that `django_settings.INSTALLED_APPS` now won't exist at the point were this code is executed. But do these paths need to be set into `INSTALLED_APPS` elsewhere now?